app/views/step_by_step_pages/publish_or_delete.html.erb

```diff
@@ -45,7 +51,7 @@
                   text: "Unschedule publishing",
                   destructive: true
                 } %>
-              <% end %>
+            <% end %>
```

Comment:
Quite confusing that this `<% end %>` is at the same level of indentation as the `<% else %>` below it! Looks a bit like the `else` on line 55 is tied with the `if` on line 45, which I don't think is the case 🤔

Reply:
oh yes, this is a mistake. This end belongs to the `form_tag`
I will re-do the whole view with Leena's approach, which should make the view more readable.